apps/camera/js/controllers/controls.js

```diff
@@ -27,10 +27,30 @@ function ControlsController(app) {
   this.activity = app.activity;
   this.createView();
   this.bindEvents();
+  this.l10nGet = app.l10nGet;
+  this.updateA11yLabels();
   debug('initialized');
 }
 
 /**
+ * Set initial a11y labels on the camera view elements
+ *
+ * @private
+ */
+ControlsController.prototype.updateA11yLabels = function() {
+  // We need the app to be first localized
+  // before setting proper aria-Labels
+  if (!this.app.localized()) {
+    return;
+  }
+
+  this.view.els.radios.camera.setAttribute('aria-label',
+    this.l10nGet('camera-mode-radio-button'));
+  this.view.els.radios.video.setAttribute('aria-label',
+    this.l10nGet('video-mode-radio-button'));
```

Comment:
This approach exposes 'private' view implementation details to the controller. I'd prefer something like:

```js
this.view.setAriaLabels({
  camera: this.l10nGet('camera-mode-radio-button'),
  video: this.l10nGet('video-mode-radio-button')
});
```